docs/pages/usage-billing.mdx

```diff
@@ -0,0 +1,88 @@
+---
+title: Usage Reporting and Billing
+description: Provides a detaield breakdown of Teleport usage reporting and billing.
+---
+
+Teleport Team, Teleport Enterprise Cloud, and Teleport Enterprise send
+anonymized usage data to Teleport so we can calculate billing metrics. This
+guide describes the anonymized usage data and the billing metrics we calculate.
+
+## Anonymized usage data
+
+The commercial editions of Teleport send anonymized information to Teleport's
+cloud infrastructure. This information contains the following:
+
+-   Teleport license identifier
+-   Anonymized cluster name and Teleport Auth Service host ID
+-   For each Teleport user, the anonymized user name and a count of interactions
+    with infrastructure resources enrolled in your Teleport cluster.
```

Comment:
How should this be corrected then?

Reply:
I'm not quite sure how we should word it - especially since usage events on Cloud are actually more extended than just the ones that we use for billing - but the gist is that in Cloud we keep an anonymized log of each individual "billing" event when it happens and then we count and sum over the billing period, whereas for on-prem (only Enterprise, currently) the Teleport cluster does a first aggregation step on a 15-min basis and we receive the pre-aggregated counts of events per user, which we then count or sum over the billing period.

Reply:
Ok, @ptgott can we just change "a count" to "an anonymized log"?

Reply:
How important is it to customers that we describe the precise nature of the data, i.e., whether we aggregate it on the Teleport Cloud infra vs. whether it comes pre-aggregated from the customer's Teleport deployment? If it's important to specify, it looks like "anonymized log" is only accurate for Teleport Cloud. 

I'll suggest:

```markdown
- For each Teleport user, the anonymized user name and a count of interactions
 with infrastructure resources enrolled in your Teleport cluster. 

Self-hosted Teleport deployments aggregate interaction data before it reaches Teleport Cloud. 
Teleport Cloud accounts provide this information as an anonymized log, which 
Teleport aggregates over the billing period.
```